app/src/main/res/layout/continue_this_thread_view.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ * Copyright 2009 Andrew Shu
```

Comment:
This file is copyright you rather than Andrew Shu.

You can do one of three things to this line:
 - delete it (and the following blank)
 - update it to Copyright 2019 your name
 - update it to Copyright 2019 Diode Contributors

Reply:
> Works great for me +1
> 
> Is it possible to have the back button go back to the comments, and preferably the same previous position, as with the "Click here to view the rest of the comments" button?

I'll look into it.